autoscheduler/scraper/models/course.py

```diff
@@ -1,4 +1,18 @@
 from django.db import models
 
-class Course(models.Model):
-    """ Describe model here """
+class Course(models.Model): # Directly from GBS:w
+    id = models.CharField(max_length=10, primary_key=True) # Could be dept+course_num
+    dept = models.CharField(max_length=4, db_index=True) # CSCE
+    course_num = models.CharField(max_length=5, db_index=True) # i.e. 221
+    title = models.CharField(max_length=100) # Course title, i.e. "Data Structres & Algorithms"
+    description = models.TextField(null=True, blank=True)
+    prerequisites = models.TextField(null=True, blank=True)
+    corequisites = models.TextField(null=True, blank=True)
+    cd = models.CharField(max_length=3)#yes or no. states whether course is a cd credit
+    icd = models.CharField(max_length=3)#yes or no. states whether course is an icd credit

```

Comment:
If these are "yes" or "no", shouldn't they instead be a [BooleanField](https://docs.djangoproject.com/en/2.2/ref/models/fields/#booleanfield)?

Reply:
Not necessarily but they can be. Considering there are only two options in this situation, that's a better idea. I shall make the change.